### DISCUSSION TITLE
Where should the Native files get created?

### DISCUSSION THREAD

**Discussion by trevski** [dlemstra/Magick.NET#1232] [Help]
Getting consistent 

Unable to load DLL 'Magick.Native-Q8-x64.dll': The specified module could not be found. (Exception from HRESULT: 0x8007007E)

Error message when trying to debug in Visual Studio web project.

Where should the Native files get created ? could it be a permissions issue, do I need to install something else.

**Comment by dlemstra** [comment]
The native library should be copied to your `runtimes` or `bin` folder but that all depends on which platform / target framework you are using.

**Comment by trevski** [comment]
OK we are running a dlls in a web project from the local /bin folder

We have Magic.Net-Q8-x64.dll  and  Magic.Net.Core.dll in the bin but no Magick.Native-Q8-x64.dll, can you provide any insight into why this might not be being created. 

Both installed from NuGet

runtimes are in the packages folder in the project but not the execution path.

It works if I manually copy in the runtime, is this a required step ? It was not clear in the documentation

**Reply by dlemstra** [reply]
On which operating system are you running this project?

**Reply by trevski** [reply]
Windows 11 on the dev environment and Windows Server 2016 on deployment

**Reply by dlemstra** [reply]
That's odd because the binaries should be copied. Is it possible to create a super small sample project that reproduces this issue?
